lib/matplotlib/axis.py

```diff
@@ -2186,10 +2201,9 @@ def set_default_intervals(self):
     def get_tick_space(self):
         ends = self.axes.transAxes.transform([[0, 0], [1, 0]])
         length = ((ends[1][0] - ends[0][0]) / self.axes.figure.dpi) * 72
-        tick = self._get_tick(True)
         # There is a heuristic here that the aspect ratio of tick text
         # is no more than 3:1
-        size = tick.label1.get_size() * 3
+        size = self._get_tick_label_size('x', True) * 3
```

Comment:
```suggestion
        size = self._get_tick_label_size('x', major=True) * 3
```
For better readability.

Reply:
Or since this is a private helper, you could simplify to `_get_major_tick_label_size()`. We can always add minor when needed.

Reply:
OK, I removed `major` since we don't really use it.